plugins/notifications/src/components/NotificationsPage/NotificationsPage.tsx

```diff
@@ -45,13 +46,16 @@ export const NotificationsPage = () => {
   const [sorting, setSorting] = React.useState<SortBy>(
     SortByOptions.newest.sortBy,
   );
+  const [severity, setSeverity] =
+    React.useState<NotificationSeverity>('normal');
```

Comment:
I think this should be empty by default and show all notifications and user can then filter the severity he/she wants to see if necessary? Otherwise they might miss some notifications

Reply:
It's like log-level elsewhere - the `debug` messages are not shown by default. 

I do not think this is an issue but if the concern stays, we can show `low` by default. It will be effectively the same.